src/Test.elm

```diff
@@ -99,26 +105,26 @@ describe untrimmedDesc tests =
     in
         if desc == "" then
             Internal.failNow
-                { description = "You must pass 'describe' a nonempty string!"
+                { description = "This `describe` has an empty description string. Let's give it a nonempty one!"
```

Comment:
Same comment about empty strings. (Also because of trimming, blank is more accurate than empty.)